data/translations.de.json

```
@@ -207,9 +207,9 @@
     "VulnerabilitiesItemsAcknowledgmentsTitle": "Danksagungen bzgl. der Schwachstelle",
     "VulnerabilitiesItemsAcknowledgmentsDescription": "Enthält eine Liste von Danksagungen, die mit dieser Schwachstelle in Verbindung stehen.",
     "VulnerabilitiesItemsCveTitle": "CVE",
-    "VulnerabilitiesItemsCveDescription": "Enthält die MITRE-Standardnummer für die CVE-Verfolgung (Common Vulnerabilities and Exposures) der Schwachstelle.",
+    "VulnerabilitiesItemsCveDescription": "Enthält die MITRE CVE-Nummer (Common Vulnerabilities and Exposures) der Schwachstelle.",
     "VulnerabilitiesItemsCweTitle": "CWE",
-    "VulnerabilitiesItemsCweDescription": "Enthält die MITRE-Standardaufzählung der Schwachstellen (Common Weakness Enumeration, CWE) für die betreffende Schwachstelle.",
+    "VulnerabilitiesItemsCweDescription": "Enthält die MITRE CWE-Nummer (Common Weakness Enumeration) der Schwachstelle.",
```

It is not only the number, so maybe:
```suggestion
    "VulnerabilitiesItemsCweDescription": "Enthält die MITRE CWE-Kategorisierung (Common Weakness Enumeration) der Schwachstelle.",